pkgs/applications/graphics/photoflow/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchFromGitHub, gettext, glib, libxml2, pkgconfig, swig, automake, gobjectIntrospection, cmake, ninja, libtiff, libjpeg, fftw, exiv2, lensfun, gtkmm2, libraw, lcms2, libexif, vips, expat, pcre, pugixml }:
```

Comment:
Depends on https://github.com/darktable-org/rawspeed, could you package it as well and use that instead of the bundled one?

Reply:
Hmm, apparently it is not possible at the moment. I opened an issue https://github.com/aferrero2707/PhotoFlow/issues/175